src/core/src/op/divide.cpp

```diff
@@ -208,26 +193,25 @@ bool evaluate_bound(const Node* node, ov::TensorVector& output_values, bool is_u
 
         // replace values where zeros were found in the second argument to maximum values
         OPENVINO_SUPPRESS_DEPRECATED_START
-        auto output_maximum_value = get_constant_max_of_type(output_values[0].get_element_type());
+        const auto output_maximum_value = ngraph::get_constant_max_of_type(output_values[0].get_element_type());
```

Comment:
```suggestion
        const auto output_maximum_value = ov::util::make_tensor_of_max_value(output_values[0].get_element_type());
```
Then tensor creation from const will  be not required below.

Reply:
Done